frontend/libs/core/src/env/environment.ts

```diff
@@ -3,4 +3,5 @@ import { Environment } from './environment.interface';
 export const environment: Environment = {
   production: false,
   pocketbaseUrl: 'http://127.0.0.1:8090',
+  version: '1.0.0',
```

Comment:
Zou van het versie nummer hier `dev` maken. Anders moeten we bij een versie update het versie nummer op 2 plaatsen aanpassen.